components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java

```diff
@@ -321,19 +321,23 @@ public void updateRoleName(int tenantId, String oldRoleName, String newRoleName)
             throw new IdentitySCIMException("Error when updating role name of the role: " + oldRoleName);
         }
     }
+
     /**
-     * Lists the Groups created from SCIM with a attribute filter and search regex
+     * Method to list the groups created from SCIM with a attribute filter and search regex.
```

Comment:
Better to start with `list...` without mentioning the method.

Reply:
Fixed: 0811c8c408e0d35596e71d82f167868b3f079ae7